### PR TITLE
Init

### PR DESCRIPTION
**Pull request by rhpvorderman**

### CODE REVIEW

File: wiseguy.wdl [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+
+import "tasks/common.wdl" as common
+import "tasks/wiseguy.wdl" as wiseguy
+import "tasks/samtools.wdl" as samtools
+import "tasks/bedtools.wdl" as bedtools
+
+workflow wiseguyCnv {
+    input {
+    # IndexedBamFile from common.wdl
+    Array[IndexedBamFile] referenceBams
```
Comment (DavyCats) [inline wiseguy.wdl]: Indentation

**Review by DavyCats — CHANGES_REQUESTED** [review]
The term reference seems to be getting used in two ways, which might get confusing. Perhaps using case/control would be clearer then sample/reference?

**Comment by rhpvorderman** [comment]
> The term reference seems to be getting used in two ways, which might get confusing. Perhaps using case/control would be clearer then sample/reference?

:+1:

**Review by DavyCats — APPROVED** [review]

File: example_inputs.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
```
Comment (sndrtj) [inline example_inputs.json:1]: As a general comment, I don't know much about Wdl, but is this a schema of some form? If so, what about a json schema? That seems more generic to me.

File: wiseguy.wdl [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+    call samtools.BgzipAndIndex as wiseguyReferenceBgzip {
+        input:
+            type = "bed",
+            outputDir = outputDir,
+            inputFile = bedtoolsSortRef.bedFile
+    }
+
+
+    # Prepare sample
+    call wiseguy.Count as wiseguyCountCase {
```
Comment (sndrtj) [inline wiseguy.wdl]: wiseguy became wisestork in release.

File: xhmm.wdl [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+import "tasks/common.wdl" as common
+
+workflow XHMM {
+    input {
+        File referenceFasta
```
Comment (sndrtj) [inline xhmm.wdl:27]: this seems incomplete?

**Review by sndrtj — CHANGES_REQUESTED** [review]
Wiseguy (renamed to wisestork) has an official release now. This workflow still points to an outdated version:

https://github.com/biowdl/tasks/blob/73f621b1c7a77fd0a3c2e7498804190a1d94e37c/wiseguy.wdl

File: example_inputs.json (hunk 2 above)
Comment (rhpvorderman) [reply example_inputs.json:1]: No this is just an example. Documentation etc.

File: wiseguy.wdl (hunk 3 above)
Comment (rhpvorderman) [reply wiseguy.wdl]: Yes :P Will rename everything.

File: xhmm.wdl (hunk 4 above)
Comment (rhpvorderman) [reply xhmm.wdl:27]: Yes. It was not needed last time. It can be kept as a stub.

**Pull request merged**